src/eventHandlers/checkSuite/index.ts

```diff
@@ -19,7 +19,7 @@ const getCommitHeadline = (): string => {
   return commitHeadline;
 };
 
-export const handle = async (octokit: GitHub): Promise<void> => {
+export const checkSuiteHandle = async (octokit: GitHub): Promise<void> => {
   const pullRequests = context.payload.check_suite.pull_requests;
 
   for (const pullRequest of pullRequests) {
```

Comment:
We should add a check for the `check_suite` conclusion:
`checkSuite.conclusion === 'success'`

Reply:
Added an issue to tackle this https://github.com/ridedott/auto-merge-action/issues/17